packages/common/src/configuration/service-configuration.ts

```diff
@@ -94,7 +101,24 @@ export class ServiceConfiguration {
                     doc: 'Timeout value after which the task has to be terminated',
                 },
             },
-
+            jobManagerConfig: {
+                targetQueuedTasksOverloadRatio: {
```

Comment:
> targetQueuedTasksOverloadRatio [](start = 15, length = 31)

may be rename to queuedTaskOverloadRatio or queuedToRunningTaskRatio?

Reply:
Renamed to activeToRunningTasksRatio